mara_db/views.py

```diff
@@ -24,7 +24,7 @@ def navigation_entry():
                 uri_fn=lambda current_db=alias: flask.url_for('mara_db.index_page', db_alias=current_db))
             for alias, db in config.databases().items()
             if (isinstance(db, dbs.PostgreSQLDB) and not isinstance(db, dbs.RedshiftDB))
-               or isinstance(db, dbs.MysqlDB)  # for now, only show postgres and mysql schemas
```

Comment:
Can we make this an extra singledispatch function so that one can do:

```py
if supports_extract_schema(db)
```

This would actually allow external packages supply the needed functions instead of always needing to change it here.

Reply:
Or even easier: 

```py
supported = list(extract_schema.registry.keys())
supported.remove(object)
...
if isinstance(db, supported)
...
```

Reply:
@jankatins agree with you; didn't like it either but tried to make the pull-request as small as possible so it is easy to merge.
now it's added